### PR TITLE
Feat/navigate from Dashboard card  to Account

### PR DESCRIPTION
**Pull request by cripacrip**

### CODE REVIEW

File: src/components/page-sections/dashboard/AccountsList/AccountsList.vue [hunk 1]
```diff
@@ -4,7 +4,10 @@
       v-for="account in allAccounts"
       :key="account.id"
     >
-      <Account :account="account" />
+      <Account
+        :account="account"
+        @click="$router.push({ path: '/account', query: { id: account.accountId, type: 'mono' } })"
```
Comment (letehaha) [inline src/components/page-sections/dashboard/AccountsList/AccountsList.vue:9]: Let's move this code to a separate method in the `methods` list. Make the same thing in the `Accounts.vue` file, where you did take this code.
Also, move `type: 'mono'` to the constants in the `js/const` as a new file `accounts.const.js` and variable name `ACCOUT_TYPES`. Add a few account types: `mono` and `custom`.

**Review by letehaha — CHANGES_REQUESTED** [review]
Merge your branch with `master` to delete 3 useless commits which you took from another branch.

File: src/components/page-sections/dashboard/AccountsList/AccountsList.vue [hunk 2]
```diff
@@ -4,7 +4,10 @@
       v-for="account in allAccounts"
       :key="account.id"
     >
-      <Account :account="account" />
+      <Account
+        :account="account"
+        @click="redirectToAccount"
```
Comment (letehaha) [inline src/components/page-sections/dashboard/AccountsList/AccountsList.vue:9]: You need to pass an argument to the function here

File: src/pages/Accounts.vue [hunk 3]
```diff
@@ -44,7 +44,7 @@
         <div
           class="accounts__item"
           :class="{ 'accounts__item--disabled': !account.isEnabled }"
-          @click="$router.push({ path: '/account', query: { id: account.accountId, type: 'mono' } })"
+          @click="redirectToAccount"
```
Comment (letehaha) [inline src/pages/Accounts.vue:47]: You need to pass an argument to the function here

File: src/components/page-sections/dashboard/AccountsList/AccountsList.vue [hunk 4]
```diff
@@ -50,6 +53,9 @@ export default {
     ...mapActions('bankMonobank', {
       fetchAccounts: bankMonobankVuexTypes.FETCH_ACCOUNTS,
     }),
+    redirectToAccount(account) {
+      this.$router.push({ path: '/account', query: { id: account.accountId, type: 'ACCOUNT_TYPES.mono' } });
```
Comment (letehaha) [inline src/components/page-sections/dashboard/AccountsList/AccountsList.vue:57]: `type: ACCOUNT_TYPES.mono`, not `type: 'ACCOUNT_TYPES.mono'`. You need to pass variable value, but not the variable name lol

File: src/pages/Accounts.vue [hunk 5]
```diff
@@ -124,6 +124,9 @@ export default {
         },
       });
     },
+    redirectToAccount(account) {
+      this.$router.push({ path: '/account', query: { id: account.accountId, type: 'ACCOUNT_TYPES.mono' } });
```
Comment (letehaha) [inline src/pages/Accounts.vue:128]: `type: ACCOUNT_TYPES.mono`, not `type: 'ACCOUNT_TYPES.mono'`. You need to pass variable value, but not the variable name

**Review by letehaha — CHANGES_REQUESTED** [review]

File: src/components/page-sections/dashboard/AccountsList/AccountsList.vue [hunk 6]
```diff
@@ -22,6 +26,9 @@ export default {
   components: {
     Account,
   },
+  data: () => ({
+    ACCOUNT_TYPES,
```
Comment (letehaha) [inline src/components/page-sections/dashboard/AccountsList/AccountsList.vue:30]: You don't need to add any value to this object until it is not used in the html-part

File: src/pages/Accounts.vue [hunk 7]
```diff
@@ -76,8 +76,12 @@ import {
 } from '@/store';
 import { formatAmount } from '@/js/helpers';
 import { MODAL_TYPES } from '@/components/Modal';
+import { ACCOUNT_TYPES } from '@/js/const';
 
 export default {
+  data: () => ({
+    ACCOUNT_TYPES,
```
Comment (letehaha) [inline src/pages/Accounts.vue:83]: You don't need to add any value to this object until it is not used in the html-part

**Review by letehaha — CHANGES_REQUESTED** [review]

File: src/components/page-sections/dashboard/AccountsList/AccountsList.vue (hunk 6 above)
Comment (cripacrip) [reply src/components/page-sections/dashboard/AccountsList/AccountsList.vue:30]: Understood, I delete it

File: src/pages/Accounts.vue (hunk 7 above)
Comment (cripacrip) [reply src/pages/Accounts.vue:83]: Understood, I delete it

**Review by letehaha — APPROVED** [review]

**Pull request merged**